sandbox-web/src/main/java/com/exadel/sandbox/team5/controller/DiscountRestController.java

```diff
@@ -49,5 +56,19 @@ public void delete(@PathVariable Long id) {
     public List<Review> getReviewsByDiscount(@PathVariable Long discountId) {
         return reviewService.getReviewsByDiscount(discountId);
     }
+
+    @ApiOperation("Generating QR code with text \"Exadel employee. Special discount\"")
+    @PostMapping(value = "/qrcode", produces = MediaType.IMAGE_PNG_VALUE)
```

Comment:
why not `GET` method?

Reply:
I thought that in this case it didn't matter. But I understood that it would be more correct to use Get, so I replaced it with Get.